.github/workflows/deploy-stage.yml

```diff
@@ -26,6 +26,7 @@ jobs:
     secrets: inherit
     with:
       runner_image_name: ${{ needs.build-push-docker-image.outputs.runner_image_name }}
+      is_deploy_localstack: false
```

Comment:
Did you mean should_deploy?
will_deploy?

Reply:
this variable is for if `lint-test-security` gonna deploy localstack when running `ci checks`. the localstack deploy check was available for stage deployment. but right now I'm skipping this can there is a bug with localstack for step function

Reply:
Updated the variable name